README.md

```diff
@@ -30,14 +30,29 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.map creates a new array with the function affecting the elements, while .forEach mutates the original array.
```

Comment:
`.map()` creates an array and returns it but it is up to you to assign it somewhere so it persists.  `.forEach()` does not mutate the original array, it only loops over the array using the cb.